MetaMask/metamask-extension
# UX: Multichain: Send Flow: Wire To / Your Accounts

## **Description**
Wires up the "To" field in the new Send flow with the "Your Accounts" block

## **Manual testing steps**

1.  `MULTICHAIN=1 yarn start`
2. Click the Action menu, then "Send"
3. See all accounts listed
4. Click on any account
5. See the "To" field populated with the correct information

## **Screenshots/Recordings**


https://github.com/MetaMask/metamask-extension/assets/46655/d0627114-94f4-4e2b-8326-a006b32f17a7



## **Related issues**

Fixes https://github.com/MetaMask/MetaMask-planning/issues/1447

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
